### DISCUSSION TITLE
Welcome to RTKLIB Discussions!

### DISCUSSION THREAD

**Discussion by tomojitakasu** [tomojitakasu/RTKLIB#682] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AB-Normal1960** [comment]
Hello. I was sent to this Github because I am using the RTKLIB software to feed RTCM3 data from a local online streaming service. I use that correctional data to make the measurements of my uBlox ZED-F9P receiver more accurate.
Using the RTKNAVI module, I enter the source IP address and the port. I enter my username and password they gave me. I click on get mountpoints and i do get a list of their pount points. However, it keeps timing out when I click start. Is there a way to increase the timeout?
